res/css/views/rooms/_EventTile.pcss

```diff
@@ -648,6 +648,32 @@ $left-gutter: 64px;
             margin-top: 0;
             margin-bottom: 0;
         }
+
+        /* EventTile on every layout */
+        .mx_EventTile & {
+            /* Paragraph below ul or ol */
+            :is(ul, ol) + p {
+                margin-block: $spacing-16;
+            }
```

Comment:
this seems to be directly overriding the settings at lines 647-650. Do we really need both settings?

Reply:
Are you suggesting that `ul > p` and `ol > p` would be covered by `:is(blockquote > p, ol, ul)` ?

Reply:
Erm, I'm mostly asking the same question as in the review summary:

> Generally: as I understand it, this works by setting a `margin-top` for `<p>` elements which follow a list. What is the reason for doing that, rather than setting a `margin-bottom` on lists, to bring them into line with the other block elements? I'm worried that with all these overrides, we're making the CSS very hard to reason about, and that there may be other situations that this won't fix.